assets/js/script.js

```diff
@@ -0,0 +1,178 @@
+// $(document).ready(function() {
+// })
+
+// Thing to consider is that the program uses 3 states -
+// 2 - not running
+// 1 - play mode
+// 0 - die mode - no input is accepted.
+
+
+$(function () {
+
+    // saving dom objects to variables
+  var $container = $('.container')
+  var $bird = $('.bird')
+  var pole = $('.pole')
+  var poleTop = $('#pole_1')
+  var poleBottom = $('#pole_2')
+  var score = $('#score')
+  var restartBtn = $('#restart-btn')
+  var $birdHeight = parseInt($bird.height())
+  var conWidth = parseInt($container.width())
+  var conHeight = parseInt($container.height())
+  var polePos = parseInt(pole.css('right'))
+  var pole_initial_height = parseInt(pole.css('height'))
+  var $birdLeft = parseInt($bird.css('left'))
+
+  var gameState = 2
+  var speed = 0
+  var scoreAdd = false
+  var stopKeyDown = false;
+
+  function start () {
+    birdFlap()
+  }
+
+  // audio
+  function spop() {
+    document.getElementsByClassName('startpop')[0].play()
+  }
+
+  function flapsound() {
+    // document.getElementsByClassName('birdFlapping')[0].currentTime = 1.3
+    document.getElementsByClassName('birdFlapping')[0].pause()
+    document.getElementsByClassName('birdFlapping')[0].play()
```

Comment:
You already stored some dom accessing values into variables. (e.g. line 13 ~ )
What will be the benefit of doing that, and what can you do on line 43, 43?